config/scenarioUrls.json

```diff
@@ -20,40 +20,66 @@
       "path": "ll_bg_css_single_colon"
     }
   },
+  "test:lrc": {
+    "delayJs13Live": {
+      "path": "delayjs_1-3_live_template"
+    },
+    "delayJs68Live": {
+      "path": "delayjs_68_live_template"
+    },
+    "lcp6647Live": {
+      "path": "lcp_6647_live"
+    },
+    "lcpLive4Template": {
+      "path": "lcp_live_4_template"
+    },
+    "lcpLiveTestSpan": {
+      "path": "lcp_live_test_span"
```

Comment:
Let's remove this template from e2e as it's failing cause of edge acceptable case https://wp-media.slack.com/archives/CUKB44GNN/p1727454310142779?thread_ts=1727444219.707339&cid=CUKB44GNN